pkg/pillar/scripts/device-steps.sh

```diff
@@ -7,21 +7,24 @@ WATCHDOG_PID=/run/watchdog/pid
 WATCHDOG_FILE=/run/watchdog/file
 CONFIGDIR=/config
 PERSISTDIR=/persist
+PERSIST_STATUS_DIR=/persist/status
 PERSIST_CERTS=$PERSISTDIR/certs
 BINDIR=/opt/zededa/bin
 TMPDIR=/persist/tmp
 ZTMPDIR=/var/tmp/zededa
 DPCDIR=$ZTMPDIR/DevicePortConfig
 FIRSTBOOTFILE=$ZTMPDIR/first-boot
-GCDIR=$PERSISTDIR/config/GlobalConfig
+GCDIR=$PERSISTDIR/config/ConfigItemValueMap
+LAST_IMAGE_VERSION_FILE=$PERSIST_STATUS_DIR/last-image-version
 AGENTS0="logmanager ledmanager nim nodeagent"
 AGENTS1="zedmanager zedrouter domainmgr downloader verifier identitymgr zedagent baseosmgr wstunnelclient"
 AGENTS="$AGENTS0 $AGENTS1"
 TPM_DEVICE_PATH="/dev/tpmrm0"
 PATH=$BINDIR:$PATH
 
 echo "$(date -Ins -u) Starting device-steps.sh"
-echo "$(date -Ins -u) EVE version: $(cat $BINDIR/versioninfo)"
+EVE_VERSION=$(cat $BINDIR/versioninfo)
```

Comment:
We shouldn't be using versioninfo from BINDIR -- that's been deprecated long time ago and we really should get rid of it in favor of /etc/eve-version

Reply:
I was used above.. Wanted to be consistent.. Will change both

Reply:
Done.. Changed it.